<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APT Analyzer</class>
 <widget class="QMainWindow" name="APT Analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1775</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddTechnique">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Technique</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Tactics">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="Techniques">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TacticsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="plainText">
     <string>Tactics</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TechniquesBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Techniques</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="DescriptionBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>521</width>
      <height>531</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PotentialMatches">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Potential Matches above 50%: #</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Evaluate">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>102</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteTechnique">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>540</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Technique</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SoftwareBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Software</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Software">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSoftware">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Software</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddedTechniquesText">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Added Techniques</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteSoftware">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>540</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Software</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddedSoftwareText">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Added Software</string>
    </property>
   </widget>
   <widget class="QListWidget" name="AddedTechniquesBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>321</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="AddedSoftwareBox">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>60</y>
      <width>301</width>
      <height>471</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>39</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>39</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>147</green>
          <blue>147</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAll">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>580</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="updating">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>480</y>
      <width>102</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <zorder>PotentialMatches</zorder>
   <zorder>DeleteTechnique</zorder>
   <zorder>DeleteSoftware</zorder>
   <zorder>AddedSoftwareText</zorder>
   <zorder>AddedSoftwareBox</zorder>
   <zorder>ClearAll</zorder>
   <zorder>TacticsBox</zorder>
   <zorder>TechniquesBox</zorder>
   <zorder>SoftwareBox</zorder>
   <zorder>Tactics</zorder>
   <zorder>Techniques</zorder>
   <zorder>Software</zorder>
   <zorder>AddTechnique</zorder>
   <zorder>AddSoftware</zorder>
   <zorder>DescriptionBox</zorder>
   <zorder>Evaluate</zorder>
   <zorder>AddedTechniquesText</zorder>
   <zorder>AddedTechniquesBox</zorder>
   <zorder>updating</zorder>
   <zorder>progress</zorder>
   <zorder>ResetButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1775</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionUpdate">
   <property name="text">
    <string>Update Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
